.rat-excludes

```diff
@@ -66,3 +66,8 @@ google-big-query.svg
 google-sheets.svg
 postgresql.svg
 snowflake.svg
+
+# docs-related
+docs/docs/.htaccess*
```

Comment:
doesn't seem like we should need this line... wondering what might've changed here.